js/src/forum/components/FlagPostModal.js

```diff
@@ -46,6 +46,12 @@ export default class FlagPostModal extends Modal {
                 <input type="radio" name="reason" checked={this.reason() === 'off_topic'} value="off_topic" onclick={m.withAttr('value', this.reason)}/>
                 <strong>{app.translator.trans('flarum-flags.forum.flag_post.reason_off_topic_label')}</strong>
                 {app.translator.trans('flarum-flags.forum.flag_post.reason_off_topic_text')}
+                {this.reason() === 'off_topic' ? (
+                  <div>
+                    <label>{app.translator.trans('flarum-flags.forum.flag_post.optional_details')}</label>
```

Comment:
Now the label needs to be linked to the textarea. :wink:

Reply:
Also, I'd prefer if we could move this block outside of the surrounding `<label class="checkbox">`.

Reply:
So sorry - extreme tiredness is kicking in, only 5 hours sleep in 2 days :( 

I need to take a break from this for the moment, will pick it back up over the weekend...

Reply:
@franzliedke just having a second think, rather than having a label for the textarea, to save some screen real estate, how about using the `textarea`'s `placeholder`

Example:
![image](https://user-images.githubusercontent.com/16573496/72188062-51c19f00-33f1-11ea-83fe-135898fbc7cb.png)